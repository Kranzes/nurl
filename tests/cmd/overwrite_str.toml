args = [
    "https://github.com/nix-community/nurl",
    "v0.3.0",
    "--overwrite-str",
    "rev",
    "v${version}",
]
