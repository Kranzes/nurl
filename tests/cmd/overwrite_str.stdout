fetchFromGitHub {
  owner = "nix-community";
  repo = "nurl";
  rev = "v${version}";
  hash = "sha256-jZ+cCp1THDhfHH5yMmRPjGuthOqsgcF/3OjZ61FMdA4=";
}